### ISSUE TITLE
config list 

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#11]
I've tested the `config list` command.

So I'm wondering whether the "extended" output has anything to do with "extended" file we have as alternative to keyring?

Since the info that are provided in basic and extended mode are not intersecting, maybe we can change the `--extended` flag into something like `--more`, `--extra_info`, `--supplemental_data` or `--further`?

**Comment by d5000** [comment]
> So I'm wondering whether the "extended" output has anything to do with "extended" file we have as alternative to keyring?

Yes, the `--extended` flag changes/shows settings exactly in this "extended" file. 
In this file, also addresses are stored (as alternative to keyring), alongside decks, utxos, proposals, transaction HEX strings etc.
So I would strongly favour keeping the name of the "extended" flag.

**Comment by buhtignew** [comment]
Agreed.
But were are extended and basic config files stored?

**Comment by d5000** [comment]
In Linux/Unix, in /home/USERNAME/.config/pacli.

I don't know exactly where it's stored on Windows, but guess in the AppData folder. The location is determined by the  user_config_dir variable of the "appdirs" standard library in Python. Basically, it's there where the OS normally stores the configurations of the apps.

The basic config file is pacli.conf, the extended one is extended_config.json.
